### ISSUE TITLE
Crash with dispatchTo

### ISSUE THREAD

**Issue by bmarty** [vector-im/element-android#6080]
### Steps to reproduce

Reported by rageshakes:
https://github.com/matrix-org/element-android-rageshakes/issues?q=is%3Aopen+dispatchTo+

```
Thread: main, Exception: java.util.ConcurrentModificationException
at java.util.LinkedHashMap$LinkedHashIterator.nextNode(LinkedHashMap.java:757)
at java.util.LinkedHashMap$LinkedKeyIterator.next(LinkedHashMap.java:780)
at im.vector.app.core.utils.JsonViewerStylerKt.dispatchTo(JsonViewerStyler.kt:4)
at org.matrix.android.sdk.internal.session.DefaultSession$$ExternalSyntheticLambda1.run(R8$$SyntheticClass:5)
```

### Outcome

Crash

### Will you send logs?

No

**Comment by bmarty** [comment]
Related to #5821 probably.

**Comment by ouchadam** [comment]
Fixed in 1.4.26
